packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -137,7 +135,6 @@ export type ReactFabricType = {
     callback: ?Function,
   ): any,
   unmountComponentAtNode(containerTag: number): any,
-  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: SecretInternalsFabricType,
```

Comment:
I think we still need to export an empty object here so that it matches the paper renderer. Of code reads a key off and checks if the value is defined, that would now throw in fabric right because the object would be undefined?

Reply:
The one current caller doesn't check if it's defined.

https://github.com/facebook/react-native/blob/master/Libraries/Utilities/JSDevSupportModule.js#L23-L24

And we're not going to be adding this back. No more internals.